.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,93 @@
+name: Prerelease
+
+on: 
+  ## Uncomment when ready to actually release this thing with automated tests.
+  # Run Every Wednesday at 12:00 AM UTC
+  # schedule:
+  #   - cron: '0 0 * * 3'
+  workflow_dispatch:
+
+permissions: write-all
+
+jobs:
+  prerelease:
+    name: Prerelease
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.HCP_SDK_PIPELINE_TOKEN }}
+
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          cache: true
+          go-version-file: 'go.mod'
+          cache-dependency-path: go.sum
+
+      - name: Configure Git
+        env:
+          TOKEN: ${{ secrets.HCP_SDK_PIPELINE_TOKEN }}
+        run: |
+            git config --global advice.detachedHead false
+            git config --global url."https://${TOKEN}:x-oauth-basic@github.com/".insteadOf "https://github.com/"
+            git config user.name "hashicorp-cloud"
+            git config user.email "hashicorp-cloud@hashicorp.com"
+
+      - name: Install Go Binaries
+        env:
+          GOPRIVATE: 'github.com/hashicorp/*'
+        run: |
+          go install github.com/hashicorp/go-changelog/cmd/changelog-build@6ec9be372335f39c5df27e232c3669db7f5183a5
+          go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.49.0
+
+      - name: Run Unit Tests and Linter
+        run: make test-ci
```

Comment:
Will we plug in a `testacc-ci` step here?

Reply:
Yep! Once I have it all working. It's not here yet (or even here but commented out) because I had all the testing in another file just in case something like this PR had to happen!